ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh

```diff
@@ -71,21 +71,29 @@ done
 
 # The timeout should be much lower, this is due to https://bugzilla.redhat.com/show_bug.cgi?id=2060091
 echo "Waiting for cluster to become available"
-oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${CLUSTER_NAME}
+oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${CLUSTER_NAME} || {
+  echo "Cluster did not become available"
+  oc get hostedcluster --namespace=clusters -o yaml ${CLUSTER_NAME}
```

Comment:
Is it possible to leak any sensitive information with this command?

Reply:
Should be fine, the dump does not include any secrets to the ephemeral cluster.